beacon/duty_data.go

```diff
@@ -0,0 +1,76 @@
+package beacon
+
+import (
```

Comment:
@nivBlox this should be in a proto of sorts.. seems to be a copy of something

Reply:
iv'e removed it form proto cause there is no use for it in network